### ISSUE TITLE
Inconsistency in the order of methods described in API Documentations

### ISSUE THREAD

**Issue by Nazarah** [PlatformOfTrust/docs#59]
# What is the problem?
Although it is upto the API designer how to list HTTP methods in an API performing a particular function, we should maintain some order to show which method should come first, which next...... and so on. When reading API documentations, I found out the following inconsistencies:

1. under Identity API's `/identities/{version}` path, `**GET**` is listed before `**POST**`
<img width="799" alt="Screen Shot 2019-04-12 at 10 24 58" src="https://user-images.githubusercontent.com/17976243/56019790-3add0780-5d0e-11e9-89f0-1eece23cec15.png">

2. under Product API's `/products/{version}` `**POST**` is listed before `**GET**`
<img width="517" alt="Screen Shot 2019-04-12 at 10 18 12" src="https://user-images.githubusercontent.com/17976243/56019811-46c8c980-5d0e-11e9-8fae-da995446a1c9.png">

# What do you expect to see / happen? 

[This](https://www.restapitutorial.com/lessons/httpmethods.html) is a typical convention we can adopt.
- Add something (POST)
- Display (GET)
- Update something / or replace (PUT)
- Update / modify (PATCH)
- remove something (DELETE)

<img width="1189" alt="Screen Shot 2019-04-12 at 10 33 04" src="https://user-images.githubusercontent.com/17976243/56019854-64962e80-5d0e-11e9-9aa2-3d8a829eda06.png">

In summary, whatever core APIs, Data products we are designing, we should maintain a specific order to list all methods under a specific path.

**Comment by kyyberi** [comment]
Will reordering in RAML files change this?

**Comment by soderluk** [comment]
@kyyberi: Yes, it should.

**Comment by kyyberi** [comment]
Would you @Nazarah have time to do this in Sprint 23? (ends 22.5.2019). The RAML files are in https://github.com/PlatformOfTrust/docs/tree/master/raml2markdown/src

**Comment by Nazarah** [comment]
reply to [comment](https://github.com/PlatformOfTrust/docs/issues/59#issuecomment-488947416)
Lets sit today, so that I can get a clear understanding to the work I need to do.
